src/local-fs.ts

```diff
@@ -286,11 +287,12 @@ export default class LocalFS implements ILocalFSPackageManager {
   private _createFile(name: string, contents: any, callback: Function) {
     this.logger.trace({ name }, '[local-storage/_createFile] create a new file: @{name}');
 
-    fs.exists(name, exists => {
-      if (exists) {
-        this.logger.trace({ name }, '[local-storage/_createFile] file cannot be created, it already exists: @{name}');
-
-        return callback(fSError(fileExist));
+    fs.open(name, 'wx', err => {
```

Comment:
https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_file_system_flags

`'wx' Like 'w' but fails if the path exists.`

Could you explain me why this change?

Reply:
Oh I see, nevermind, there was `fs.exists` before.